### DISCUSSION TITLE
Heat ≠ Heat

### DISCUSSION THREAD

**Discussion by TimoDiepers** [sentier-dev/sentier_vocab#4] [Missing taxonomy]
### Missing taxonomy area

heat

### Justify the need to invest resources and add complexity

There is a big difference between heat at 30°C and heat at 1500°C - there are many processes that require a certain temperature level to work at all. An extended classification would help here.

In a hierarchical / dynamic linking sense, it would be nice if one could specify that a process demands heat at 500°C, and if there is no available process directly providing that, another process providing heat at *at least 500°C* should be chosen. 

### Any other context

Could also be connected to a certain heat carrier medium.

**Comment by jakobsarthur** [comment]
As heat itself is the contained energy in a medium, the classification for  'commodity' heat should have at least a temperature range and a medium. Some ideas on classifications could be taken from [this publication](https://www.researchgate.net/publication/293013995_A_Review_-_An_Optimization_of_Macro-encapsulated_Paraffin_used_in_Solar_Latent_Heat_Storage_Unit#fullTextFileContent) (e.g. see figure 1)

**Comment by flechtenberg** [comment]
I feel this is connected to broader classification of Thermodynamics and Heat Transfer areas. Heat can be linked to temperature (ranges) as commented above. 

For instance: Currently, SKOSMOS has no concept for latent or sensible heat, nor for the mode of transfer (convection, conduction, and radiation). Found no sources for easily adding new taxonomy for this though.

**Comment by muelleram** [comment]
While we couldn't find an official classification of temperature ranges for heat, e.g. low temperature heat ranges from XZ  to XZ degrees C etc., we found a classification for heat resistance for products (t ranges: https://armadex.com/temperature-classes-and-t-ratings/
